src/components/Exercise.js

```diff
@@ -1,11 +1,11 @@
 import { addIndex, map, toUpper } from 'ramda'
 import React from 'react'
-import { View, Text, Platform } from 'react-native'
-import Sound from 'react-native-sound'
+import { View, Text } from 'react-native'
 
 import speakerImage from '../assets/images/speaker.png'
 
 import { RoundImageWithButton } from '../components/Components'
+import { getSoundWithLength } from '../components/Utils'
```

Comment:
Sure, got it. Let’s talk about how and where we want to implement these helper functions that are used by multiple components sometime next week.